kind.sh

```diff
@@ -131,16 +156,20 @@ parse_command_line() {
 
 install_kind() {
     echo 'Installing kind...'
-    curl -sSLo kind "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
-    chmod +x kind
-    sudo mv kind /usr/local/bin/kind
+
+    mkdir -p "$kind_dir"
+
+    curl -sSLo "$kind_dir/kind" "https://github.com/kubernetes-sigs/kind/releases/download/$version/kind-linux-amd64"
```

Comment:
should use $arch here too?

Reply:
We shouldn't mess with the download URL. On Ubuntu, `uname -m` return `x86_64`. We can discuss making this OS agnostic but that should be a separate PR.